presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -96,12 +96,16 @@
 
     private Optional<Insert> insert = Optional.empty();
 
-    public Analysis(Statement root, List<Expression> parameters)
+    // for describe input and describe output
+    private final boolean isDescribe;
```

Comment:
This should probably be `isDescribeInput` so it's not confused with `DESCRIBE` / `SHOW COLUMNS`